src/components/custom-field-input-multiple-choice/custom-field-input-multiple-choice.test.jsx

```diff
@@ -56,6 +57,17 @@ describe('<CustomFieldInputMultipleChoice>', () => {
         userEvent.click(screen.getByText('Choice 1'));
         expect(screen.getByLabelText('test label', { selector: 'input' })).toHaveValue('');
       });
+
+      it('has aria-haspopup for accessibility', () => {
+        render(<CustomFieldInputMultipleChoice {...requiredProps} />);
+        expect(screen.getByLabelText('test label', { selector: '[aria-haspopup="listbox"]' })).toBeInTheDocument();
+      });
+
+      it('has aria-expanded for accessibility', () => {
+        render(<CustomFieldInputMultipleChoice {...requiredProps} />);
+        userEvent.click(screen.getByText('test label'));
+        expect(screen.getByLabelText('test label', { selector: 'ul[aria-expanded=true]' })).toBeInTheDocument();
```

Comment:
@juanca is this the best way to select the element?

Reply:
If possible, I would make the selector as simple as possible but then assert something about the attribute.

```
expect(screen.getByRole('listbox')).toHaveAttribute('aria-expanded', 'true');
```